test/lib/lbt/bundle/Builder.js

```diff
@@ -321,6 +330,58 @@ ${SOURCE_MAPPING_URL}=Component-preload.js.map
 			"myModuleUsingGlobalScope.js",
```

Comment:
@flovogt: you make an even bigger point here :-( 

First of all, even if it is hard to update the aggregation, I guess it would be doable. But there's a bigger problem with sections of mode `bundleInfo`. The filters for such sections are redundant to the real bundle definition. And, when building the `bundleInfo` section, we don't look into the content but just resolve the filters based on the names. If a module matches the filters but isn't eligible for preload-bundling (e.g. uses globals), then we would still add it to the bundleInfo, although it's not contained there.

Constant solution would be to analyse modules in all cases, but that for sure has a performance penalty (if we don't add caching for that information).

Reply:
The team agreed on removing the modules / submodules information also form the bundle info in case the file content is not part of the bundle